src/controller/Controller.php

```diff
@@ -21,6 +21,8 @@ class Controller
 
     protected $languages;
 
+    protected $translator;
```

Comment:
Why does the Controller need its own translator? I think there should be only one translator, in the Model.

Reply:
Removed in new commit: https://github.com/NatLibFi/Skosmos/commit/48b31a51acb2fd8be2452c033e0cecd05d86c764